jobs/postgres_store.go

```diff
@@ -249,7 +249,8 @@ func (pq *postgreSQLStore) FindOne(index string) (*Job, error) {
 				ta.instance_id,
 				ta.public_ip,
 				ta.private_ip,
-				jobs.array_properties
+				jobs.array_properties,
+				log_group_name
```

Comment:
Shouldn't there be sql to add the log_group_name column to the jobs table?

Reply:
not sure i follow, this query is for retrieving, not for writing. the insert queries all have the log_group_name.

Reply:
But the jobs table doesn't have this column yet, right? See the migrations directory for how to add a new column.. It should default to /aws/batch/job

Otherwise, won't existing batchiepatchie installations break?

Reply:
my bad, i forgot to commit the migration